cmd/kubeadm/app/apis/kubeadm/validation/validation.go

```diff
@@ -416,23 +416,18 @@ func ValidateMixedArguments(flag *pflag.FlagSet) error {
 }
 
 func isAllowedFlag(flagName string) bool {
-	isAllowed := false
-	switch flagName {
-	case kubeadmcmdoptions.CfgPath,
+        knownFlags := sets.NewString(kubeadmcmdoptions.CfgPath,
```

Comment:
also this is not indented with a tab, but indended with spaces.
please fix that.